src/WebJobs.Script.WebHost/Helpers/FileSystemHelpers.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Helpers
+{
+    public static class FileSystemHelpers
```

Comment:
There's a `FileUtility` class already in the project. Some things here seem to overlap, but we should either move what is here there or vice-versa, but we shouldn't have both.